packages/@lwc/synthetic-shadow/src/polyfills/event/polyfill.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { defineProperties, isNull, isUndefined } from '@lwc/shared';
+import { pathComposer } from '../../3rdparty/polymer/path-composer';
+import { retarget } from '../../3rdparty/polymer/retarget';
+import { eventTargetGetter, eventCurrentTargetGetter } from '../../env/dom';
+import { eventsDispatchedDirectlyOnShadowRoot } from '../../faux-shadow/events';
+import { getShadowRoot, isHostElement } from '../../faux-shadow/shadow-root';
+import { EventListenerContext, eventToContextMap } from '../../faux-shadow/events';
+import { getNodeOwnerKey } from '../../shared/node-ownership';
+import { getOwnerDocument } from '../../shared/utils';
+
+export function targetGetter(this: Event): EventTarget | null {
```

Comment:
No need to export this function.

```suggestion
function targetGetter(this: Event): EventTarget | null {
```

Reply:
Let rename this method to `patchedTargetGetter`. The current name is ambiguous with the original getter names.